models/balancer/polygon/balancer_polygon_schema.yml

```diff
@@ -0,0 +1,54 @@
+version: 2
+
+models:
+  - name: balancer_v2_polygon_pools_fees
+    meta:
+      blockchain: polygon
+      project: balancer_v2
+      contributors: jacektrocinski
+    config:
+      tags: ['polygon', 'balancer', 'pools_fees']
+    description: >
+      Balancer v2 swap fees stored at the pool level.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_number
+            - tx_hash
+            - index
+    columns:
+      - &contract_address
+        name: contract_address
+        description: 'Polygon address for the liquidity pool used in transaction'
+        tests:
+          - not_null
+      - &tx_hash
+        name: tx_hash
+        description: ''
+        tests:
+          - not_null
+      - &index
+        name: index
+        description: ''
+        tests:
+          - not_null
+      - &tx_index
+        name: tx_index
+        description: ''
+        tests:
+          - not_null
+      - &block_time
+        name: block_time
+        description: ''
+        tests:
+          - not_null
+      - &block_number
+        name: block_number
+        description: ''
+        tests:
+          - not_null
+      - &swap_fee_percentage
+        name: swap_fee_percentage
+        description: 'Swap fees for the pool, values are returned with 18 decimals'
```

Comment:
```suggestion
        description: 'Swap fees for the pool (1e18 meaning 100%)'
```

Reply:
cc @mendesfabio wdyt? same description would have to be replicated everywhere else

Reply:
I have changed the description for `swap_fee_percentage` in all views.

This issue is now fixed.